infrastructure/main.tf

```diff
@@ -42,62 +35,7 @@ locals {
 
   users_secret_names = "${values(local.supported_users)}"
 
-  users_usernames_settings = "${zipmap(
-                                    formatlist("IDAM_USERS_%s_USERNAME", keys(local.supported_users)),
-                                    data.azurerm_key_vault_secret.idam_users_usernames.*.value
-                                )}"
-
-  users_passwords_settings = "${zipmap(
-                                    formatlist("IDAM_USERS_%s_PASSWORD", keys(local.supported_users)),
-                                    data.azurerm_key_vault_secret.idam_users_passwords.*.value
-                                )}"
-
   s2s_rg  = "rpe-service-auth-provider-${local.local_env}"
```

Comment:
why are we keeping this?

Reply:
It's used to build following vault keys:idam_users_usernames / idam_users_passwords, ie idam-users-cmc, etc